tests/by-util/test_date.rs

```diff
@@ -8,15 +8,13 @@ use rust_users::*;
 #[test]
 fn test_date_email() {
     let (_, mut ucmd) = at_and_ucmd!();
-    let result = ucmd.arg("--rfc-email").run();
-    assert!(result.success);
+    ucmd.arg("--rfc-email").succeeds();
```

Comment:
this change seem unrelated, no?

Reply:
Well kinda, since we're making parts of the struct private, and the code would've needed to be refactored, either way, I thought to do this anyway. It also makes lifetimes less of a pain on that branch.